ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -198,3 +233,309 @@ module Json = struct
   let api_errors =
     List.map (fun error -> `O [("name", `String error)]) !Api_errors.errors
 end
+
+module Types = struct
```

Comment:
I think this module could be in `CommonFunctions.ml`.

Reply:
I have moved it to `CommonFunctions.ml`.